connectors/google-maps-platform/element-templates/google-maps-platform-connector.json

```diff
@@ -0,0 +1,533 @@
+      "description": "Unstructured address. Detailes in a <a href='https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/google-maps-platform/'> connector documantation</a>",
+      "group": "input",
+      "type": "String",
+      "feel": "optional",
+      "optional": false,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "addressValue"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "validateAddress"
+        ]
+      }
+    },
+    {
+      "label": "Address",
+      "description": "Complete address. Detailes in a <a href='https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/google-maps-platform/'> connector documantation</a>",
+      "group": "input",
+      "type": "String",
+      "feel": "optional",
+      "optional": false,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "addressValue"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "getPlaceID"
+        ]
+      }
+    },
+    {
+      "label": "Destination",
+      "description": "The place ID value to which you wish to calculate directions",
+      "group": "input",
+      "type": "String",
+      "feel": "optional",
+      "optional": false,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "destinationValue"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "directions"
+        ]
+      }
+    },
+    {
+      "label": "Origin",
+      "description": "The place ID value from which you wish to calculate directions.",
+      "group": "input",
+      "type": "String",
+      "feel": "optional",
+      "optional": false,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "originValue"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "directions"
+        ]
+      }
+    },
+    {
+      "label": "Units",
+      "description": "Specifies the unit system to use when displaying results",
+      "group": "input",
+      "value": "metric",
+      "type": "Dropdown",
+      "choices": [
+        {
+          "name": "Metric",
+          "value": "metric"
+        },
+        {
+          "name": "Imperial",
+          "value": "imperial"
+        }
+      ],
+      "binding": {
+        "type": "zeebe:input",
+        "name": "unitsValue"
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "directions"
+        ]
+      }
+    },
+    {
+      "label": "Mode",
+      "description": "For the calculation of distances and directions, you may specify the transportation mode to use",
```

Comment:
replace with: You can specify the transportation mode to use when calculating distances and directions.